shell/artifacts/Social/Social.recipes

```diff
@@ -65,7 +65,7 @@ recipe
   PostTimeRanker
     input = posts
     output = rankedPosts
-  List
+  PostList
```

Comment:
What if instead there was a separate particle that sorts by `rank`, then I think you'd still be able to use `List`?

Reply:
My thought was that the input to both `List` and `Muxer` is a collection, and so technically we can't rely on the order persisting, right? Though it may be that implementation details mean that it's an array and so a `rank` sorting particle would be sufficient. I can do that if you think it's better but it felt like we should treat collections as a set.

Reply:
I see what you mean (and probably this is just my ignorance), but then how does PostTimeRanker work at all?

Reply:
I think it only works at first -- it sorts by time and sets the `rank` field in the post, and we get lucky I think when we first render the feed, but if the feed is subsequently updated through say a user adding a new post to their miniblog in another tab, I saw empirically that the feed contents were updated and the new post was not rendered in the correct order. I confirmed that with this PR the issue is fixed.

I'll look more closely to see what exactly is breaking but I assumed it was our relying on the array implementation detail and the collection update breaking that in some manner. But I see what you're saying, since PostTimeRanker rebuilds `output` each time it seems like relying on the implementation detail maybe shouldn't break.